app/javascript/dashboard/store/modules/helpCenterPortals/getters.js

```diff
@@ -0,0 +1,27 @@
+export const getters = {
+  uiFlagsIn: _state => helpCenterId => {
+    const uiFlags = _state.helpCenters.uiFlags.byId[helpCenterId];
+    if (uiFlags) return uiFlags;
+    return { isFetching: false, isUpdating: false, isDeleting: false };
+  },
+
+  isFetchingHelpCenters: _state => _state.uiFlags.isFetching,
+  helpCenterById: (...getterArguments) => helpCenterId => {
+    const [_state] = getterArguments;
+    const helpCenter = _state.helpCenters.byId[helpCenterId];
+    if (!helpCenter) return undefined;
+
+    return {
+      ...helpCenter,
+    };
```

Comment:
Can't we just return the object? This check and returning a new object seem unnecessary.